config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -2402,6 +2406,15 @@ public FeaturePolicySpec featurePolicy(String policyDirectives) {
 			return new FeaturePolicySpec(policyDirectives);
 		}
 
+		/**
+		 * Configures {@code Permissions-Policy} response header.
+		 * @param policyDirectives the policy directive(s)
+		 * @return the {@link PermissionsPolicySpec} to configure
+		 */
+		public PermissionsPolicySpec permissionsPolicy(String policyDirectives) {
```

Comment:
Similar to the feedback in `HeadersConfigurer`, this needs to be configurable via the lambda DSL.